nipap-cli/nipap

```diff
@@ -74,7 +74,12 @@ if __name__ == '__main__':
         sys.exit(1)
 
     # read configuration
-    cfg = ConfigParser.ConfigParser()
+    cfg_defaults = {
+        'scheme': 'http',
```

Comment:
Is this a "scheme"? I'm trying to think what I would intuitively call it... Isn't like SSL: on/off or True/False better and more intuitive?